playwright/_impl/_browser_type.py

```diff
@@ -235,6 +235,6 @@ def normalize_launch_params(params: Dict) -> None:
             params["ignoreAllDefaultArgs"] = True
             del params["ignoreDefaultArgs"]
     if "executablePath" in params:
-        params["executablePath"] = str(Path(params["executablePath"]))
+        params["executablePath"] = str(Path(params["executablePath"]).resolve())
     if "downloadsPath" in params:
-        params["downloadsPath"] = str(Path(params["downloadsPath"]))
+        params["downloadsPath"] = str(Path(params["downloadsPath"]).resolve())
```

Comment:
Hasn't this been fixed on the server side?

Reply:
You are right, I thought by mistake that it was on the client side.